src/foam/box/ClusterSyncRequestMessage.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.box',
+  name: 'ClusterSyncRequestMessage',
+  extends: 'foam.box.Message',
```

Comment:
We typically don't need to extend foam.box.Message, but just model your message as a standalone object and set it as the `object` property of a foam.box.Message.

I see that RPCMessage extends foam.box.Message, but I believe that is a mistake that was left in from an old refactoring.